tests/test_registration.py

```diff
@@ -52,12 +54,13 @@ def test_registration(entry_point):
     assert all_specs[1].id == "SomeEntity-v1"
 
 
-def test_register_duplicate_id():
+def test_register_duplicate_id(caplog):
     registry = registration._Registry()
 
     registry.register("Entity-v0", "fake_entry_point")
-    with pytest.raises(ValueError, match=r"Cannot re-register ID"):
+    with caplog.at_level(logging.WARNING):
```

Comment:
fyi I think you can do something like `pytest.warns`

Reply:
nice1 40f0671